src/server/utilities/extractHeaders/index.ts

```diff
@@ -8,6 +8,9 @@ const extractHeaders = (headers: IncomingHttpHeaders) => {
     isUK = headers['x-bbc-edge-isuk'] === 'yes';
   } else if (headers['x-country']) {
     isUK = headers['x-country'] === 'gb';
+  }
+
+  if (headers['x-country']) {
     showCookieBannerBasedOnCountry = COUNTRIES_WITH_COOKIE_BANNER.includes(
       headers['x-country'].toString().toLowerCase(),
     );
```

Comment:
I think we can just remove the `else`, and it should work
```suggestion
  }
  if (headers['x-country']) {
    isUK = headers['x-country'] === 'gb';
    showCookieBannerBasedOnCountry = COUNTRIES_WITH_COOKIE_BANNER.includes(
      headers['x-country'].toString().toLowerCase(),
    );
  }
```

Reply:
Can we also have a unit test to ensure code coverage?

Reply:
Good suggestion. We already had one, but it didn't take into account some extra complexities that belfrage told me about (like what was actually coming from the traffic managers etc). Done now.